spec/classes/document_search_spec.rb

```diff
@@ -199,14 +199,13 @@ def create_documents(document_hashes)
       end
 
       it 'logs details about the query' do
-        expect(Rails.logger).to receive(:error).with(/"query":"uh oh"/)
+        expect(Rails.logger).to receive(:error)
         document_search.search
       end
 
       it 'sends the error to NewRelic' do
-        expect(NewRelic::Agent).to receive(:notice_error).with(
-          error, options: { custom_params: { indices: ['test-i14y-documents-agency_blogs'] } }
-        )
+        allow(NewRelic::Agent).to receive(:notice_error).and_return(nil)
```

Comment:
Same as above... move the `allow` up into the before, then we can change this to:

```
     document_search.search
     
     expect(NewRelic::Agent).to have_received(:notice_error)
```